tools/systemd-units/login-rathena.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Rathena Login Server
+After=network-online.target remote-fs.target nss-lookup.target
+Wants=network-online.target
+#You might wanna edit the next line if your units are running in different servers instead of all in a single server.
+Requires=char-rathena.service map-rathena.service
+
+[Service]  
+Type=single
```

Comment:
It may be just me, but I really like to setup some quota to my process to ensure the host still responsive.
Now it may depend load but I think login server is pretty small. 

I'd add those in [Service] section, (require systemd >231 tough)
MemoryHigh=400M
MemoryMax=500M